arlo_server/routes.py

```diff
@@ -304,6 +305,13 @@ def serialize_members(audit_board):
     return members
 
 
+def isoformat(datetime: Optional[datetime.datetime]) -> Optional[str]:
```

Comment:
YES I was thinking the same thing

Reply:
I miss TypeScript: `file.processing_started_at?.isoformat()` FTW.